### PR TITLE
feat(manager) : Consolidating Documents

### PR DESCRIPTION
**Pull request by kotwani2883**
## Explain the feature/fix

Consolidating all the documents of SPAship at a single place

## Does this PR introduce a breaking change

No

## Screenshot(s)
Screenshot from 2023-02-08 10-42-15

### CODE REVIEW

**Review by arkaprovob — APPROVED** [review]
Looks good to me, @soumyadip007 please review this one more time before merging.

File: packages/manager/src/views/DocumentsPage/DocumentsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
+import { Gallery, GalleryItem, PageSection, Title } from '@patternfly/react-core';
+import { Banner } from '@app/components';
+import { DocumentCard } from './components/DocumentCard/DocumentCard';
```
Comment (akhilmhdh) [inline packages/manager/src/views/DocumentsPage/DocumentsPage.tsx:3]: `./components/DocumentCard`

File: packages/manager/src/views/DocumentsPage/DocumentsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+export const DocumentsPage = (): JSX.Element => {
+  const gettingStartedData = [
```
Comment (akhilmhdh) [inline packages/manager/src/views/DocumentsPage/DocumentsPage.tsx:6]: Keep all these data out side the component

**Review by akhilmhdh — CHANGES_REQUESTED** [review]

**Review by soumyadip007 — APPROVED** [review]
Looks Good to me

File: packages/manager/src/views/DocumentsPage/DocumentsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      </Gallery>
+    </PageSection>
+    <Title headingLevel="h1" style={{ marginLeft: '100px', marginTop: '20px' }}>
+      All about SPAship
+    </Title>
+    <PageSection isCenterAligned isWidthLimited className="pf-u-px-3xl">
+      <Gallery hasGutter>
+        {aboutAllData.map(({ title, linkhere, footer }) => (
+          // eslint-disable-next-line react/jsx-key
+          <GalleryItem>
```
Comment (akhilmhdh) [inline packages/manager/src/views/DocumentsPage/DocumentsPage.tsx:113]: Kindly avoid disabling eslint and pass keys

**Review by akhilmhdh — CHANGES_REQUESTED** [review]

**Review by soumyadip007 — APPROVED** [review]

**Review by akhilmhdh — APPROVED** [review]

**Pull request merged**